src/postprocessors/DagMCPostprocessor.C

```diff
@@ -0,0 +1,43 @@
+/********************************************************************/
+/*                  SOFTWARE COPYRIGHT NOTIFICATION                 */
+/*                             Cardinal                             */
+/*                                                                  */
+/*                  (c) 2021 UChicago Argonne, LLC                  */
+/*                        ALL RIGHTS RESERVED                       */
+/*                                                                  */
+/*                 Prepared by UChicago Argonne, LLC                */
+/*               Under Contract No. DE-AC02-06CH11357               */
+/*                With the U. S. Department of Energy               */
+/*                                                                  */
+/*             Prepared by Battelle Energy Alliance, LLC            */
+/*               Under Contract No. DE-AC07-05ID14517               */
+/*                With the U. S. Department of Energy               */
+/*                                                                  */
+/*                 See LICENSE for full restrictions                */
+/********************************************************************/
+
+#ifdef ENABLE_DAGMC
+
+#include "DacMCPostprocessor.h"
```

Comment:
```suggestion
#include "DagMCPostprocessor.h"
```

Reply:
Thanks for pointing this out! I fixed this locally but forgot to push it. This branch should be up-to-date now, but it requires you to do some special maneuvering with the DAGMC submodule, since I'm still waiting on [this fix to DAGMC](https://github.com/svalinn/DAGMC/pull/846) to go in.

### The special maneuvering:

As long as you don't use MOOSE's conda environment (Pyne needs to [fix this](https://github.com/pyne/pyne/pull/1469) first), you just need to check out a different DAGMC branch:

```
cd cardinal
git remote add april https://github.com/aprilnovak/cardinal.git
git fetch april
git checkout -b dagmc april/dagmc
./scripts/get-dependencies.sh

cd contrib/DAGMC
git remote add april https://github.com/aprilnovak/DAGMC.git
git fetch april
git checkout -b update-pyne april/update-pyne

cd ../../
./contrib/moose/scripts/update_and_rebuild_petsc.sh
./contrib/moose/scripts/update_and_rebuild_libmesh.sh
make -j8
```

Hopefully I'll have the fixes needed for conda soon, but that relies on DAGMC and Pyne developers to merge some things.

Reply:
hmm I didn't need to remake petsc/libmesh when using the moose conda env, but I did find the make part finicky, `dagmc` appears to always want to remake even if it was built successfully, and the next time around it fails to configure. Removing the build and then re-making "fixes" that. Probably on the dagmc side or just osx wonkiness.

Tutorial runs though!

Reply:
Good to know, maybe I'm having some osx issues as well or I need to tweak the build some more. 

Glad to hear the tutorial works!